pkg/tfvars/gcp/gcp.go

```diff
@@ -3,7 +3,6 @@ package gcp
 import (
 	"encoding/json"
 	"fmt"
-
```

Comment:
I believe this blank line should remain, it intentionally separates the openshift imports from the ones above.